sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -161,10 +161,10 @@ def get_estimated_size_and_observables(self, value, nested=False):
       return self.estimate_size(value, nested), []
 
 
-class DeterministicPickleCoderImpl(CoderImpl):
+class DeterministicCoderImpl(CoderImpl):
```

Comment:
Hmm well this is still a CoderImpl so do you mean DeterministicAdaptorCoderImpl?